eth2/beacon/types/validators.py

```diff
@@ -27,7 +27,7 @@
 )
 
 
-def _round_down_to_previous_multiple(amount: int, increment: int) -> int:
+def round_down_to_previous_multiple(amount: int, increment: int) -> int:
```

Comment:
1. What do you mean by "multiple" here?
2. It seems good to use:
```suggestion
def round_down_to_previous_multiple(amount: Gwei, increment: Gwei) -> Gwei:
```
since the names of the variable `amount` and `increment` imply they are in balance units.

Reply:
1. it "rounds `amount` down to the previous multiple of `increment`"

e.g. in terms of 1 ETH increment, it goes 1 ETH ~> 1 ETH, 1.1 ETH ~> 1 ETH, 1.9 ETH ~> 1 ETH, etc.

happy to update the name if there is a clearer one

2. given how this function is used across the codebase, i was aiming to skip an additional use of `Gwei` just to satisfy `mypy`. there are two uses at the moment and they both go on to additional math -- if you are concerned about the type safety i would think it is better to just encapsulate the `effective_balance` calculation (which is what this is part of anyway and imo may be a better refactoring here)